uint8 STATUS_INITIALIZING = 0
uint8 STATUS_INITIALIZED  = 1
uint8 STATUS_ARMING       = 2
uint8 STATUS_ARMED        = 3
uint8 STATUS_INGAME       = 4
uint8 STATUS_POSTGAME     = 5
uint8 STATUS_HALT         = 6
uint8 game_status


float32 game_time_duration
float32 game_elapsed_time
float32 game_time_left
bool game_finished